spec/views/users_show_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_relative '../rails_helper'
+
+RSpec.describe 'Users', type: :system do
+  before do
+    @user1 = User.create(name: 'Alice Wonderland', photo: 'https://placehold.co/200x200',
+                         bio: "Hello, I'm Alice Wonderland", posts_counter: 5)
+    @user2 = User.create(name: 'Bob Wonderland', photo: 'https://placehold.co/200x200', bio: "Hello, I'm Bob Wonderland")
+    @post1 = @user1.posts.create(title: 'Amazing Adventures', text: 'Lorem ipsum')
+    @post2 = @user1.posts.create(title: 'Exploring Wonderland', text: 'Dolor sit amet')
+    @comment = Comment.create(text: 'Great post!', author: @user2, post: @post1)
+  end
+
+  it 'displays user information on the User show page' do
+    visit user_path(id: @user1.id)
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_content(@user1.bio)
+    expect(page).to have_selector("img[src='#{@user1.photo}']")
+    expect(page).to have_content('Total posts: 2')
+  end
+
+  it 'displays post information on the User show page' do
+    visit user_path(id: @user1.id)
+    expect(page).to have_content(@post1.title)
+    expect(page).to have_content(@post1.text)
+    expect(page).to have_content(@post2.title)
+    expect(page).to have_content(@post2.text)
+    expect(page).to have_content('See all posts')
+  end
```

Comment:
Just [as suggested by the previous reviewer](https://github.com/rmiyoyo/blog-app/pull/11#discussion_r1455461342)😉, Kindly add a test that asserts that you can see the user's first 3 posts in the `user_show` page as this is a requirement for this milestone😉.